src/render.jl

```diff
@@ -24,13 +24,25 @@ end
 
 ## 2 arg form render for comparing
 function render(mode::BeforeAfter, reference, actual)
+    if displayable(MIME("image/png"))
+        render(MIME("image/png"), mode, reference, actual)
+    else
+        render(MIME("text/plain"), mode, reference, actual)
+    end
+end
+function render(::MIME"text/plain", mode::BeforeAfter, reference, actual)
     println("- REFERENCE -------------------")
     render_item(mode, reference)
     println("-------------------------------")
     println("- ACTUAL ----------------------")
     render_item(mode, actual)
     println("-------------------------------")
 end
+function render(::MIME"image/png", mode::BeforeAfterImage, reference, actual)
+    println("- REFERENCE --------|--------- ACTUAL -")
+    display(MIME("image/png"), mosaicview(reference, actual; nrow=1, npad=5))
```

Comment:
I thought `MIME("image/png")` or `"image/png"` would work, but this still renders images both in atom plot panel and in terminal 😕